back/engines/commercial/nlp/app/services/nlp/api.rb

```diff
@@ -151,15 +155,39 @@ def text_network_analysis(tenant_id, project_id, locale, max_nodes: nil, min_deg
       response = post(
         "/v2/tenants/#{tenant_id}/project/#{project_id}/ideas/text_network_analysis",
         body: body.to_json,
-        headers: { 'Content-Type' => 'application/json' }
+        headers: authorization_header.merge('Content-Type' => 'application/json')
       )
 
       raise ClErrors::TransactionError.new(error_key: response['code']) unless response.success?
 
       response.parsed_response.dig('data', 'task_id')
     end
     # rubocop:enable Metrics/MethodLength
+
+
+    # @param [String] tenant_id
+    # @param [String] text
+    # @param [String] locale
```

Comment:
Shall we add a `@return [something?]` here so we also communicate what the method will return?

Reply:
I must admit I have a very liberal use of Yard annotations :p Depending on how much work it is to get the it right and the value it brings, I sometimes add some hinting. We could also add `@raise` annotations but that's really a matter of how far we want to go there. Maybe something we should discuss with the team.

In this case, I'm entirely sure what's returned except that it's a `Hash`. I don't know the type of the keys (probably strings) and the values.